### PR TITLE
[feat] Homepage

### PR DESCRIPTION
**Pull request by shivampandey0**
- Created Header Component
- Created Footer Component
- Featured Categories Components
- Homepage Component

### CODE REVIEW

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+              aria-label="Search Products"
+            />
+          </div>
+          <div className="nav mobile-nav">
+            <button className="btn nav-close-btn">
+              <i className="fas fa-arrow-left" />
+            </button>
+            <ul className="menu">
+              <li>
+                <a href="./pages/login.html" className="btn btn-primary">
```
Comment (ArjunGTX) [inline src/components/header/Header.jsx]: `Link` can be used instead of `a` to avoid page refresh

**Review by ArjunGTX — APPROVED** [review]
Code looks clean and well structured. Keep up the good work.

File: src/components/header/Header.jsx (hunk 1 above)
Comment (shivampandey0) [reply src/components/header/Header.jsx]: Thanks for noticing, will fix it.

File: public/index.html [hunk 2]
```diff
@@ -24,6 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
```
Comment (Hamdrive) [inline public/index.html]: Cleanup the commented text generated from MockBee boilerplate

File: src/pages/home/CategoryLink.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import styles from "./Home.module.css";
+import { Link } from "react-router-dom";
+
+export const CategoryLink = ({ categoryImg, categoryName }) => {
+  return (
+    <>
+      <Link to="/products" className="flex-column flex-center">
+        <img  src={categoryImg} alt="" className={styles.home_featured} />
```
Comment (Hamdrive) [inline src/pages/home/CategoryLink.jsx]: Write a meaningful alt text to ensure accessiblity

File: src/pages/home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+export const Home = () => {
+  const [categories, setCategories] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const res = await axios.get("api/categories");
+        setCategories(res.data.categories);
+      } catch (error) {}
```
Comment (Hamdrive) [inline src/pages/home/Home.jsx]: As you have used a `try-catch` block, why not log the error if it does happen?

**Review by Hamdrive — COMMENTED** [review]
Great work done, learnt some good practices which I will also adopt into my project.
Have made some suggestions that can easily be refactored.

**Pull request merged**